app.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Kitty Items",
+  "description": "A complete NFT marketplace built with Cadence, Flow's resource-oriented smart contract programming language.",
+  "repository": "https://github.com/onflow/kitty-items",
+  "logo": "https://cdn.shopify.com/s/files/1/1061/1924/products/Happy_cat_emoji_icon_87ab6b39-5f08-48a5-a89a-1f5d3312e159_large.png?v=1571606090",
```

Comment:
Here's a little icon you can use in place of the Shopify link in case it gets taken down.

I would just add it to this repo and then grab the raw URL from the GitHub repo browser. (Click on file > Download > Copy URL)

![kitty-items-icon](https://user-images.githubusercontent.com/2547035/113519867-d8592380-9543-11eb-9bb3-c45e51360ccf.png)

Reply:
Awesome.